pkg/analysis/passes/discoverability/discoverability.go

```diff
@@ -0,0 +1,42 @@
+package discoverability
+
+import (
+	"encoding/json"
+
+	"github.com/grafana/plugin-validator/pkg/analysis"
+	"github.com/grafana/plugin-validator/pkg/analysis/passes/metadata"
+)
+
+var (
+	emptyDescription = &analysis.Rule{Name: "empty-description", Severity: analysis.Warning}
+	emptyKeywords    = &analysis.Rule{Name: "empty-keywords", Severity: analysis.Warning}
+)
+
+var Analyzer = &analysis.Analyzer{
+	Name:     "discoverability",
+	Requires: []*analysis.Analyzer{},
+	Run:      run,
+	Rules:    []*analysis.Rule{},
```

Comment:
You forgot to add the rules here :smile: 

```suggestion
	Rules:    []*analysis.Rule{emptyDescription, emptyKeywords},
```